Fix the pkg-config version and architecture-dependent library directory.

--- a/sdl2pp.pc.in
+++ b/sdl2pp.pc.in
@@ -1,11 +1,11 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include
 
 Name: sdl2pp
 Description: C++ bindings/wrappers for SDL2
-Version: @SDL2PP_VERSION@
+Version: @PROJECT_VERSION@
 Requires: @SDL2_ALL_PKGCONFIG_MODULES@
 Conflicts:
 Libs: @SDL2PP_EXTRA_PKGCONFIG_LIBRARIES@ -L${libdir} -lSDL2pp
